test/e2e/config/caren.yaml

```diff
@@ -194,9 +202,9 @@ variables:
   # # Disable Prism Central certificate checking. Default: false
   # NUTANIX_INSECURE: false
   # # Prism Central user
-  NUTANIX_USER: ""
+  #  NUTANIX_USER: ""
```

Comment:
Please revert.

Reply:
as discussed offline, we should fix this in CAPX infrastructure-components.yaml. This should not be a requirement for clusterctl init.